### PR TITLE
Updated LICENSE and a small fix in kwargs

### PR DESCRIPTION
**Pull request by mauro3**

### CODE REVIEW

**Comment by tlycken** [comment]
:+1:

I find it fantastic that we can finally leave the somewhat hairy mess of having code based on proprietary software behind us. Great job!

File: LICENSE.md [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
+> EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+> MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+> NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+> LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+> OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+> WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+The `ode4`, `ode4s`, `ode4ms` routines were implemented by Patrick O'Leary.
+The `ode23s` functions were implemented by Alexander Croy.
```
Comment (acroy) [inline LICENSE.md]: Minor nitpick: it is just one function :-)

Comment (mauro3) [inline LICENSE.md]: Done. Also removed ode4 which is gone.

Comment (acroy) [inline LICENSE.md]: Thanks!

**Pull request merged**